test/test_unistd.rs

```diff
@@ -78,8 +79,8 @@ fn test_mkstemp() {
 
 #[test]
 fn test_getpid() {
-    let pid = getpid();
-    let ppid = getppid();
+    let pid: ::libc::pid_t = getpid().into();
```

Comment:
You're right that it can't fail, but it can return 0. I meant to suggest using an `Option<Pid>` type here rather than `Result`. The thing is that the user shouldn't need to compare the `Pid` to 0, that's should be something abstracted away by this newtype. So if the user gets a `Pid` it should be valid, otherwise they should get a `None`.